### DISCUSSION TITLE
Deal with RGB/RGBA images

### DISCUSSION THREAD

**Discussion by jeaunrg** [Hboni/francis#54] [Ideas]
1) add modules to extract one channel or convert gray images to rgb
2) some images are RGBA but alpha is 255 everywhere: **load only RGB channels**
3) some images are RGB but each channel has the same value (pseudo-gray images): **load only one channel**
4) do not apply thresholding, morphomath and operations on alpha channel

For now, none of our model functions handle rgb and rgba images.

**Comment by Hboni** [comment]
As discussed yesterday, I agree with your propositions 👍
